esmvaltool/utils/recipe_test_workflow/recipe_test_workflow/opt/rose-suite-metoffice.conf

```diff
@@ -0,0 +1,7 @@
+[template variables]
+DRS_CMIP6="BADC"
+DRS_OBS4MIPS="default"
+MODULE_NAME="scitools/community/esmvaltool/2.6.0"
+ROOTPATH_CMIP6="/project/cma/esmvaltool/CMIP6"
+ROOTPATH_OBS4MIPS="/project/cma/esmvaltool/obs"
```

Comment:
Update these paths to align with the paths in, e.g. https://github.com/ESMValGroup/ESMValCore/pull/1709

Reply:
9aac67af5 done.